### DISCUSSION TITLE
Downgrading

### DISCUSSION THREAD

**Discussion by Crazygird** [3kh0/ext-remover#425] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi, just wondering if I was to powerwash my chromebook when I downgrade, would my computer be unenrolled from my district? I want to downgrade but I also don't want to disconnected from my school.

**Comment by rwg100** [comment, marked as answer]
dont downgrade, its illegal anyways, and you will get in trouble anyways

**Comment by obkduishinijf** [comment]
probably not, unless you user shimmer (mark as answer)
